MOONCLOCK/apps.py

```diff
@@ -66,10 +68,10 @@ def update(self, first, remaining_duration):
 
         if self.show_seconds:
             string = '{}  {}  {}'.format(
-                str_rjust(str(hours), 2, '0'), str_rjust(str(minutes), 2, '0'), str_rjust(str(seconds), 2, '0'))
+                str_align(str(hours), 2, '0', self.align), str_align(str(minutes), 2, '0', self.align), str_align(str(seconds), 2, '0', self.align))
```

Comment:
Here we should use `str_rjust` because we always want to add zeros to the left.

Reply:
What about this @gorrdy?

Reply:
now it should be ok